deepchecks/tabular/checks/model_evaluation/boosting_overfit.py

```diff
@@ -229,6 +232,19 @@ def condition(result: dict):
         name = f'Test score over iterations is less than {format_percent(threshold)} from the best score'
         return self.add_condition(name, condition)
 
+    def config(self, include_version: bool = True) -> 'CheckConfig':
+        """Return check instance config."""
+        if self.alternative_scorer is not None:
+            for k, v in self.alternative_scorer.items():
+                if not isinstance(v, str):
+                    name = type(self).__name__
+                    raise ValueError(
+                        f'Serialization of "{name}" check instance is not supported '
```

Comment:
Who sees this message? In case it's the user, I suggest making it clearer what was wrong and what they should be doing. For example "Only recognized string scorers are allowed when sterilizing checks. For a list of recognized strings please refer to https://docs.deepchecks.com/en/stable/user-guide/general/metrics_guide.html#list-of-supported-strings"

Reply:
I am not against rephrasing error messages, so yeap will do that.

But I am not sure about links in the error message/ User always can do `help(CheckType)` to get a docstring of a check without going anywhere

Reply:
@yromanyshyn notice the link that @noamzbr sent is not tied directly to the check, so `help(CheckType)` won't get you there that easily

Reply:
I rephrased error messages and added references to the docs